saleor/plugins/avatax/plugin.py

```diff
@@ -616,6 +606,72 @@ def calculate_order_shipping(
             gross=price,
         )
 
+    def calculate_order_shipping(
+        self, order: "Order", previous_value: TaxedMoney
+    ) -> TaxedMoney:
+        if self._skip_plugin(previous_value):
+            return previous_value
+
+        if not charge_taxes_on_shipping():
+            return previous_value
+
+        if not _validate_order(order):
+            return previous_value
+
+        taxes_data = get_order_tax_data(
+            order, self.config, self.config.tax_included, False
+        )
+
+        return self._calculate_order_shipping(order, taxes_data, previous_value)
+
+    def calculate_order_total(
```

Comment:
Based on `Codecov` labels from below, it looks like the tests for this method are missing 🤔